style.css

```diff
@@ -419,6 +443,56 @@ main {
   align-items: center;
 }
 
+/* Modals & Overlay */
+.modal {
+  position: fixed;
+  top: 1rem;
+  left: 1rem;
+  right: 1rem;
+  border-radius: 16px;
+  z-index: 10;
+  background-color: #fff;
+  height: 95%;
+  overflow-y: auto;
+  transform: scale(0);
+  transition: 300ms ease-in-out;
+}
+
+.modal.active {
+  transform: scale(1);
+  transition: 300ms ease-in-out;
+}
+
+.modal .work_card {
+  width: 100%;
+  max-width: 100%;
+}
+
+.modal .action {
+  justify-content: flex-start;
+  gap: 12px;
+  padding-top: 25px;
+  margin-top: 18px;
+  border-top: 1px solid #ebecf0;
+}
+
+.overlay {
+  position: fixed;
+  opacity: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 5;
+  background-color: rgba(103, 199, 208, 0.5);
```

Comment:
- [ ] This background color is very different from the design. See the difference.

### Your design 

![Screenshot from 2021-07-01 20-59-49](https://user-images.githubusercontent.com/22434660/124150715-701cce00-daaf-11eb-8407-a7281a31ad8a.png)


### Figma design

![Screenshot from 2021-07-01 21-00-10](https://user-images.githubusercontent.com/22434660/124150744-78750900-daaf-11eb-8342-1014cfd5b689.png)